Form field component ttl picker not initially enabling

zofskeez opened the pull request:
The _TtlPicker2_ instance in the _FormField_ component was missing the `initialEnabled` argument which was causing the toggle to be unchecked and the value to be hidden initially. Toggling the input would reveal the set value.
![image](https://user-images.githubusercontent.com/24611656/142138040-ed45e262-a68e-4fbf-9511-3a775d7ec23f.png)
After adding the argument the initial state is properly reflected. 
![image](https://user-images.githubusercontent.com/24611656/142138063-b4fbaad0-b31f-4549-ac32-bb57f0399231.png)

Monkeychip reviewed (approved):
Nice work. Remember the labels (UI) and then a milestone label of 1.9.1

zofskeez commented:
Thanks @Monkeychip. After looking into your comment on Slack I noticed that the toggle was enabled for the default `0s` value which I fixed and added a test.
